src/pages/requests/RequestDetailsPage.tsx

```diff
@@ -127,6 +108,7 @@ export const RequestDetailsPage = () => {
               Megtartom
             </Button>
           )}
+          {(isOwner || isAdmin) && <RequestActions requestId={request.id} deleteRequest={deleteRequest} />}
```

Comment:
deleteRequest doesn't have to be defined in RequestDetailsPage and then passed down, you can just use the hook in RequestActions